src/main/java/com/ubb/zenith/model/UserPlaylist.java

```diff
@@ -0,0 +1,63 @@
+package com.ubb.zenith.model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserPlaylist {
+    @Embeddable
+    public static class UserPlaylistsId implements Serializable {
+        private Integer userId;
+        private Integer playlistId;
+
+        // Default constructor
+        public UserPlaylistsId() {}
```

Comment:
Since you use @NoArgsConstructor annotation, you don't have to write the default constructor because this annotations is responsible for this